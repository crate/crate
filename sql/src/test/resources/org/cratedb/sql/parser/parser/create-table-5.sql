create table test.t1 (col1 integer not null, primary key (col1))