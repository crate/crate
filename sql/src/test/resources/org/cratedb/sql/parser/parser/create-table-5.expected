org.cratedb.sql.parser.parser.CreateTableNode@7f5663a2
name: test.t1
statementType: CREATE TABLE
lockGranularity: R
existenceCheck: NO_CONDITION
tableElementList: 	
	org.cratedb.sql.parser.parser.TableElementList@a761fe

	[0]:	
	org.cratedb.sql.parser.parser.ColumnDefinitionNode@6d126b07
	type: INTEGER NOT NULL
	name: col1
	elementType: null
	[1]:	
	org.cratedb.sql.parser.parser.ConstraintDefinitionNode@5430d082
	constraintName: null
	constraintType: PRIMARY_KEY
	properties: null
	name: null
	elementType: AT_DROP_CONSTRAINT
	columnList: 		
		org.cratedb.sql.parser.parser.ResultColumnList@50c931fc

		[0]:		
		org.cratedb.sql.parser.parser.ResultColumn@48f0c0d3
		exposedName: col1
		name: col1
		tableName: null
		isDefaultColumn: false
		type: null